misk-crypto/README.md

```diff
@@ -0,0 +1,59 @@
+This module provides Misk with the ability to store and manage cryptographic key material in a
+safe and easy way.
+
+How it works?
+-----
+The `Cipher` object is used to encrypt/decrypt data.
+
+It uses a key which is loaded from a configuration file at startup.
+The key associated with each cipher should be stored **encrypted** in the app's common configuration file.
+
+In order to initialize the module, the app must have a configured GcpKmsClient (Google),
+or an AwsKmsClient (Amazon) client in order to have access to the KMS.
+
+Setup
+-----
+In one of your app's module files:
+```$kotlin
+class MyAppModule : KAbstractModule {
+  override fun configure() {
+    val kmsClient = AwsKmsClient().withDefaultCredentials()
```

Comment:
Can we make this provided by an AwsKmsClientModule or a GcpKmsClientModule, rather than asking the client to pass it in?

Reply:
That's a good point, and also something I've been battling with since I started writing this module.
As I see it, the client must provide some kind of configuration/object that says "hi this where my KMS is, here's how to access it" - be it AWS credentials or GCP (or even custom thing, IDK).
We could probably automate and wrap this with an `AwsKmsClientModule` or something, but having that makes a few assumptions about the environment (like if there's an environment variable or a file in a certain path containing credentials)

I feel like it'd be a good addition to provide some kind of way to automatically get some an object, but I think it'd be better to put it in our internal misk implementation.
Because:
1. As I said, I want this to be env-agnostic as possible
2. We'll probably have better tools and infrastructure around this to make it super easy to integrate with AWS.